cmd/kola/options.go

```diff
@@ -93,6 +93,11 @@ func init() {
 	sv(&kola.ESXOptions.Server, "esx-server", "", "ESX server")
 	sv(&kola.ESXOptions.Profile, "esx-profile", "", "ESX profile (default \"default\")")
 	sv(&kola.ESXOptions.BaseVMName, "esx-base-vm", "", "ESX base VM name")
+
+	// oci-specific options
+	sv(&kola.OCIOptions.Region, "oci-region", "", "OCI region")
+	sv(&kola.OCIOptions.Image, "oci-image-id", "", "OCI image id")
```

Comment:
I meant `OCI image ID` for the help text, not the parameter name.  We don't use an `-id` suffix for any other parameters.